web/main/colors.css

```diff
@@ -24,30 +24,28 @@
       inset 2px 0 0 var(--land-color),
       5px 0 0 var(--land-color),
       -5px 0 0 var(--land-color);
+    --navbar-shadow: 0 -1px 0 oklch(from var(--shadow-color) l c h / 8%);
```

Comment:
What do you think if we made token more universal?

It is not just for navbar only but for any floating component. Something like `--float-shadow`.

Reply:
It's a good idea! I've changed the name